### ISSUE TITLE
Feature Request: Support predefined prompts

### ISSUE THREAD

**Issue by kepi** [chieapp/chie#18]
I believe that having predefined prompts significantly enhances the user experience when interacting with ChatGPT and other engines. I would like the following features to be added:

1. be able to define and manage prompts
2. use prompt in any chat, i.e. using slash commands (like in https://github.com/lencx/ChatGPT)
3. import prompts from some sites, like https://prompts.chat/

Last one can come any time later, or maybe just leave it to some feature plugin.

**Comment by zcbenz** [comment]
Not a complete solution, but if you use OpenAI API backend, you can set system prompt in the assistant settings.
<img width="462" alt="Screenshot 2023-08-20 at 8 29 00" src="https://github.com/chieapp/chie/assets/639601/7449ec0c-c41a-43d0-973f-ea24988d31a9">

**Comment by kepi** [comment]
Yeah, I know about system prompt. But that would be really lot of work to use :)

**Comment by skorokithakis** [comment]
I agree with @kepi. My idea of predefined prompts would be to perhaps have "template" conversations, e.g. I could have one called "Spanish conversation", where my prompt would be something like "help me improve my Spanish by talking to me about everyday things and point out/correct mistakes I made". Then, when I wanted to have a conversation like that, I would select "New chat -> Spanish conversation", and it would open a conversation with that prompt already entered and responded to.
